### PR TITLE
Move information from README to documentation

### PR DESCRIPTION
**Pull request by padix-key**
This PR collects information previously residing in the README and puts it into the documentation, primarily into the *Contributor guide*.

### CODE REVIEW

**Comment by naefl** [comment]
Hmmm, I personally wouldn't do this. I think 1) many, if not most, people prefer not having to interact with the python api and just take the files from the buckets 2) the current python install still doesn't work well on macOS due to openstructure 3) every extra link people have to click to get started is extra friction.

I think we should leave the current getting started, my understanding is this new [getting started](https://github.com/plinder-org/plinder/blob/836c57e88449c5dc4e1060b5f62ae830785d77fa/README.md#-getting-started) won't even work outside of docker, so until that is resolved I don't think we should advertise the python api much.

Agree with moving the the dev instructions

**Comment by padix-key** [comment]
I actually agree. My proposal is more about moving more detailed section, such as the workflow description, into the docs. The *Getting started* section now mentions the Google Bucket, but the Python installation instructions still needs improvement. This will be covered in an upcoming PR this week, where `plinder` is released on PyPI.

**Comment by naefl** [comment]
Sounds good - right now the `Getting Started` in the `README.md` AFAICT removes instructions of how to use the data from the bucket and leaves the python installation instructions - I assume you'll invert that later (i.e. do the opposite: remove python, add back the instructions how to use the data from the bucket)

**Comment by padix-key** [comment]
I readded the dataset download instructions

File: README.md [hunk 1]
```diff
@@ -17,273 +17,90 @@
...
-    pip install .
+The *PLINDER* project is a community effort, launched by the University of Basel,
+SIB Swiss Institute of Bioinformatics, VantAI, NVIDIA, MIT CSAIL, and will be regularly
+updated.
 
-Or with a development installation:
+To accelerate community adoption, PLINDER will be used as the field’s new Protein-Ligand
+interaction dataset standard as part of an exciting competition at the upcoming 2024
+[Machine Learning in Structural Biology (MLSB)](https://mlsb.io) Workshop at NeurIPS,
+one of the fields’ premiere academic gatherings.
```
Comment (yusuf1759) [inline README.md:41]: nit; change fields' to field's

File: README.md [hunk 2]
```diff
@@ -17,273 +17,90 @@
...
+[documentation website](https://plinder-org.github.io/plinder/).
 
-# 📃 Publications
+# 📃 Citation
 
 Durairaj, Janani, Yusuf Adeshina, Zhonglin Cao, Xuejin Zhang, Vladas Oleinikovas, Thomas Duignan, Zachary McClure, Xavier Robin, Gabriel Studer, Daniel Kovtun, Emanuele Rossi, Guoqing Zhou, Srimukh Prasad Veccham, Clemens Isert, Yuxing Peng, Prabindh Sundareson, Mehmet Akdel, Gabriele Corso, Hannes Stärk, Gerardo Tauriello, Zachary Wayne Carpenter, Michael M. Bronstein, Emine Kucukbenli, Torsten Schwede, Luca Naef. 2024. “PLINDER: The Protein-Ligand Interactions Dataset and Evaluation Resource.”
 [bioRxiv](https://doi.org/10.1101/2024.07.17.603955)
 [ICML'24 ML4LMS](https://openreview.net/forum?id=7UvbaTrNbP)
+
+PLease see the [citation file](CITATION.cff) for details.
```
Comment (yusuf1759) [inline README.md:104]: nit; change "PLease" to "Please"

File: README.md [hunk 3]
```diff
@@ -17,273 +17,90 @@
...
-|   |-- ligand_scores        # ligand similarity parquet dataset
-|   |-- ligands              # ligand data expanded from entries for computing similarity
-|   |-- linked_structures    # linked apo and predicted structure files
-|   |-- links                # parquet files linking systems to apo and predicted structures
-|   |-- mmp                  # ligand matched molecular pairs (MMP) and series (MMS) data
-|   |-- scores               # protein similarity parquet dataset
+```console
+$ export PLINDER_RELEASE=2024-06 # Current release
+$ export PLINDER_ITERATION=v2 # Current iteration
+$ gsutil -m cp -r gs://plinder/${PLINDER_RELEASE}/${PLINDER_ITERATION}/* ~/.local/share/plinder/${PLINDER_RELEASE}/${PLINDER_ITERATION}/
```
Comment (yusuf1759) [inline README.md:78]: I would add a statement after this that says something like "For details on the description of sub-directories, see ..."

**Review by tjduigna — APPROVED** [review]

**Pull request merged**